crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -198,8 +201,8 @@ fn main() -> anyhow::Result<()> {
     let args: Vec<_> = args.collect();
 
     match test_mode {
-        TestMode::Node => node::execute(module, &tmpdir, &args, &tests)?,
-        TestMode::Deno => deno::execute(module, &tmpdir, &args, &tests)?,
+        TestMode::Node => node::execute(module, &tmpdir, &args, &tests, coverage)?,
+        TestMode::Deno => deno::execute(module, &tmpdir, &args, &tests, coverage)?,
```

Comment:
I can't really review Node or Deno stuff, so I will be pulling in another maintainer when we get there.

Reply:
I think this is the only thing missing.